phpunit.xml.dist

```diff
@@ -18,11 +18,11 @@
         </testsuite>
     </testsuites>
 
-    <filter>
-        <whitelist>
+    <coverage processUncoveredFiles="true">
```

Comment:
This change is not necessary and is not good because we still support phpunit 5 ;)

Reply:
Yeah, I'm trying to get the PHP 8 build to pass first. I'll revert everything that's unnecessary of course :thu

Reply:
We had this issue on Silverstripe and I addressed this by creating sminnee/phpunit, a replacement that ships 5.7.30 with PHP8 support.

Ditto for sminnee/phpunit-mock-objects

You're welcome to use it if you want.

Reply:
Thanks @sminnee, I'll take a look at everything on Monday 👍

Reply:
Reverted this. And implemented sminnee/phpunit, thanks!